src/sandstorm/run-bundle.c++

```diff
@@ -742,7 +742,7 @@ public:
   }
 
   kj::MainBuilder::Validity status() {
-    changeToInstallDir();
+    changeToInstallDirAndCheckAccess();
```

Comment:
Arguably `status` shouldn't require this check. It can often complete just fine without any write permissions.